analyses/aggregated_maps_categorical.qmd

```diff
@@ -0,0 +1,50 @@
+---
+title: "aggregated_maps_categorical"
+format: html
+---
+
```

Comment:
This should probably be called with an external run function, so it's in a separate file. Once the larger PR is approved, I can break that up into sensible runs.